src/pages/user/login/components/Login/index.tsx

```diff
@@ -122,19 +110,31 @@ class Login extends Component<LoginProps, LoginState> {
     }
   };
 
+  public static Tab = LoginTab;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Declaration of public static field not allowed after declaration of public instance method. Instead, this should come at the beginning of the class/interface.](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4526098)